api/peripheral_api/routes/Printer.js

```diff
@@ -88,4 +91,30 @@ router.post('/sendPrintRequest', async (req, res) => {
   });
 });
 
+router.post('/pushDiscordPDF', async (req, res) => {
+  const { apiKey, fileURL } = req.body;
+
+  if(apiKey !== DISCORD_PRINTING_KEY){
+    return res.sendStatus(UNAUTHORIZED);
+  }
+
+  const sqs = new AWS.SQS({apiVersion: '2012-11-05'});
+
+  const accountId = ACCOUNT_ID;
+  const queueName = PAPER_PRINTING_QUEUE_NAME;
+
+  const sqsParams = {
+    MessageBody: JSON.stringify({ fileURL }),
+    QueueUrl: `https://sqs.us-west-2.amazonaws.com/${accountId}/${queueName}`
+  };
+
+  sqs.sendMessage(sqsParams, (err, data) => {
```

Comment:
we should use [`SceSqsApiHandler`](https://github.com/SCE-Development/Core-v4/blob/2c5fd8b7b546d90a09d9be8d29b7e3b8054b0afa/api/peripheral_api/util/SceSqsApiHandler.js#L10)

open an issue for it if were not doing it rn